images/newrelic/tests/main.tf

```diff
@@ -0,0 +1,115 @@
+terraform {
+  required_providers {
+    oci  = { source = "chainguard-dev/oci" }
+    helm = { source = "hashicorp/helm" }
+  }
+}
+
+variable "digests" {
+  description = "The image digests to run tests over."
+  type = object({
+    infrastructure-bundle   = string
+    fluent-bit-output       = string
+    k8s-events-forwarder    = string
+    prometheus-configurator = string
+    kube-events             = string
+    prometheus              = string
+    kubernetes              = string
+  })
+}
+
+data "oci_string" "ref" {
+  for_each = var.digests
+  input    = each.value
+}
+
+# This is a pretty cursed method of getting the license key directly from the
+# environment, without needing to plumb through TF_VAR_
+data "external" "license-key" {
+  program = ["bash", "-c", "echo \"{\\\"output\\\": \\\"$NEW_RELIC_LICENSE_KEY\\\"}\""]
+}
+
+resource "helm_release" "nri-bundle" {
+  name       = "nri-bundle"
+  namespace  = "default"
+  repository = "https://helm-charts.newrelic.com"
+  chart      = "nri-bundle"
+
+  values = [
+    <<EOF
+global:
+  cluster: test
+  licenseKey: ${data.external.license-key.result.output}
+
+newrelic-infrastructure:
+  images:
+    # forwarder:
+    #   registry:
+    #   repository:
+    #   tag:
```

Comment:
Is this not needed because `nri-kube-events.images` does this below?

Reply:
fixed! good catch